localgov_menu_link_group.module

```diff
@@ -9,6 +9,60 @@ declare(strict_types = 1);
 
 use Drupal\localgov_menu_link_group\MenuLinkGrouper;
 use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Menu\MenuTreeParameters;
+
+/**
+ * Implements hook_preprocess_menu().
+ */
+function localgov_menu_link_group_preprocess_menu(&$variables) {
+  $variables['menu_name'] = $variables['menu_name'] ?? 'admin';
```

Comment:
I was going to query setting default menu name to admin, turns out it doesn't even matter what menu name gets passed, just needs a string and it still works :-).